components/Table.js

```diff
@@ -63,14 +69,14 @@ export default function Table({
     setAllFilters(filters);
   }, [filters]);
 
-  const handleRowClick = row => () => {
-    // Call the prop - onRowClick
-    onRowClick(row);
-  };
+  // const handleRowClick = row => () => {
+  //   // Call the prop - onRowClick
+  //   onRowClick(row);
+  // };
```

Comment:
Commented out for now until we got a design update on how to handle this. cc @mkanoria